doc/parser_list.rst

```diff
@@ -0,0 +1,130 @@
+Parser list
```

Comment:
This file is generated, it shouldn't be tracked in Git.

I should fix the script: I moved the content at the end of doc/parser.rst. Maybe I should try to use an include, or something like that?